### ISSUE TITLE
Error while using gcn_bifuse_ie

### ISSUE THREAD

**Issue by icocoder** [graph4ai/graph4nlp#487]
## ❓ Questions and Help

Hi, when I tried to run the summarization example with the gcn_bifuse_ie config, I got this warning message for lots of input files thus the vocab & data files couldn't be produced. [link](https://github.com/graph4ai/graph4nlp/blob/e7ad93f590b9399034d3ebfbd857cc5a4de977f0/examples/pytorch/summarization/cnn/config/gcn_bifuse_ie.yaml#L1)

The warning is this, and I tried different timeout thresholds (1500, 15000, 50000, etc): 
![Capture](https://user-images.githubusercontent.com/94281906/151053883-5f644131-543e-4a22-b16e-7a9b83ff5808.JPG)

**Comment by Nicoleqwerty** [comment]
Hi @icocoder, some data is indeed lost during the data processing of this task, which is limited by StanfordNLP, and we ended up retaining 29,356 of the 30,000 training set data. You can try running the code to see if only a small portion of the data is skipped. If you do not want to data loss, you can truncate the original text during preprocessing.

**Comment by icocoder** [comment]
@Nicoleqwerty Thanks a lot.
